src/arc.rs

```diff
@@ -80,9 +80,9 @@ impl Arc {
         }
     }
 
-    /// Converts an Arc into a series of cubic bezier segments.
+    /// Converts an `Arc` into a series of cubic bezier segments.
     ///
-    /// Closure will be invoked for each segment.
+    /// The closure `p` will be invoked for each segment.
```

Comment:
```suggestion
    /// The closure `p` will be invoked with the control points for each segment.
```
(Guessing what the arguments are, but users will also need to do so)

Reply:
Applied.